doc/data/system.md

```diff
@@ -6,8 +6,8 @@ A system should contain system properties, input frame properties, and labeled f
 
 ID       | Property                | Raw file     | Required/Optional    | Shape                    | Description
 -------- | ----------------------  | ------------ | -------------------- | -----------------------  | -----------
-type     | Atom type indexes       | type.raw     | Required             | Natoms                   | Integers that start with 0
-type_map | Atom type names         | type_map.raw | Optional             | Ntypes                   | Atom names that map to atom type, which is unnecessart to be contained in the periodic table
+type     | Atom type indexes       | type.raw     | Required             | Natoms                   | Integers that start with 0. If the training parameter {ref}`type_map <model/type_map>` is set and `type_map.raw` is provided, type indexes map to `type_map.raw`; otherwise, the system type indexes map to model type indexes (whether {ref}`type_map <model/type_map>` is set or not)
```

Comment:
Please check: If both `model/type_map` and `type_map.raw` are provided, the system atom type will be map to model type.

Reply:
> the system atom type will be map to model type

Isn't `type.raw` mapped to `type_map.raw`?